java/src/main/java/ai/rapids/cudf/ColumnView.java

```diff
@@ -2287,6 +2287,14 @@ public final ColumnVector getMapValue(Scalar key) {
     return new ColumnVector(mapLookup(getNativeView(), key.getScalarHandle()));
   }
 
+  public final ColumnVector listContains(Scalar key) {
```

Comment:
Can we get some java docs for what this API does?

Reply:
Added